lib/api/2.0/lookups.js

```diff
@@ -0,0 +1,101 @@
+// Copyright 2016, EMC Inc.
+
+'use strict';
+var injector = require('../../../index.js').injector;
+var controller = injector.get('Http.Services.Swagger').controller;
+var waterline = injector.get('Services.Waterline');
+
+/**
+ * @api {get} /api/2.0/lookups GET /?q=
+ * @apiVersion 2.0.0
+ * @apiDescription get list of lookups
+ * @apiName lookups-get
+ * @apiParam {query} q Query term to lookup by node, macAddress, or ipAddress.
+ * @apiGroup lookups
+ * @apiSuccess {json} lookups List of all lookups or if there are none an empty object.
+ */
+var lookupsGet = controller(function (req) {
+    return waterline.lookups.findByTerm(req.swagger.query.q);
+});
+
+/**
+ * @api {post} /api/2.0/lookups POST /
+ * @apiVersion 2.0.0
+ * @apiDescription create a lookup
+ * @apiName lookups-post
+ * @apiParam {String} macAddress MAC Address for the Lookup Record
+ * @apiParam {String} ipAddress IP Address associated with the MAC Address
+ * @apiParam {String} node Node associated with the Lookup Record
+ * @apiGroup lookups
+ * @apiSuccess {json} lookup the lookup that was created
+ */
+
+var lookupsPost = controller({success: 201}, function (req) {
+    return waterline.lookups.create(req.swagger.params.body.value);
```

Comment:
Mongo will reject it.  We setup the unique index in on-core to prevent the duplicates from being created.